app/web/templates/create_incident.html

```diff
@@ -128,9 +147,12 @@
     function maintenanceCheck(elem) {
     if (elem.value == 0) {
         document.getElementById("maintenance_end").style.display = "block";
+        document.getElementById("maintenance_desc").style.display = "block";
     } else {
```

Comment:
Isn't that reverse logic? Display, if `!elem.value`?

Reply:
elem.value in this case refers to the incident impact. 0 is maintenance. So the additional fields (description and end_date) are only displayed in case the user selected 'scheduled maintenance' from the dropdown menu.